src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -149,7 +149,7 @@ struct sdm_sensor_info
     std::array<std::string, 6> sname_end = {"label", "input", "max", "average", "highest", "status"};
     int max_end_types = sname_end.size();
     std::string errmsg, str_op, target_snode;
-    data_type data = { 0 };
+    data_type data;
```

Comment:
is this happening due to initializing string members also to zero? I didn't quite understand the fix.

Reply:
I agree with this fix,  data_type has default constructor that initializes data members appropriately,. `= {0}` only makes sense for fundamental types.  The PR comment doesn't exactly match but I get it, thanks!

Reply:
This deserves a comment.

Reply:
= {0} doesn't make sense for this structure type, but this structure is getting filled up below this code. I just wanted to understand is there any other potential bug in the code.

Reply:
I think the proper fix is 
`data_type data {};`

Reply:
> I think the proper fix is `data_type data {};`

Yes, this change is worked.